.github/workflows/build.yaml

```diff
@@ -89,27 +84,22 @@ jobs:
           name: rpm-${{ matrix.shortcut }}
           path: ${{ env.ARTIFACTS_DIR }}
 
-      - name: Install documentation generation dependencies
-        if: ${{ matrix.shortcut == 'cs9' }}
-        run: pip3 install pdoc
-
       - name: Install created sdk
-        if: ${{ matrix.shortcut == 'cs9' }}
         run: pip3 install . -U
 
       - name: Checkout target repository
-        if: ${{ matrix.shortcut == 'cs9' }}
         uses: actions/checkout@v2
```

Comment:
Do we really needed to checkout gh-pages branch also for Sphinx? We needed it for pdoc, because we had some pdoc template used for generation on gh-pages.

If really needed for Sphinx, thenp lease change to following to allow running only on CS8
```
        if: ${{ matrix.shortcut == 'cs8' }}
```